app/src/main/kotlin/dev/patrickgold/florisboard/lib/snygg/value/SnyggMaterialYouValue.kt

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2021 Patrick Goldinger
```

Comment:
For new files the copyright year should be the current, so 2023:

```suggestion
 * Copyright (C) 2023 Patrick Goldinger
```